0000-boost-qualifiers.md

```diff
@@ -0,0 +1,65 @@
+# Adjust Service Provider Boost Qualifiers
+
+- Author(s): @madninja 
+- Start Date: 2024-12-31
+- Category: technical, economic
+- Original HIP PR: <!-- leave this empty; maintainer will fill in ID of this pull request -->
+- Tracking Issue: <!-- leave this empty; maintainer will create a discussion issue -->
+- Vote Requirements: <!-- veHNT Holders, veIOT Holders, or veMOBILE Holders -->
+
+## Summary
+
+This HIP proposes adjusting the Service Provider boost eligibility qualifiers in a boosted hex to be based solely on the ongoing unique connections specified in HIP-134. This change would eliminate the ability for a hotspot in a boosted hex to qualify for boosted rewards based on Call Detail Records (CDR) in HIP-133 or by passing data (HIP-84). A hotspot would earn normal (non-boosted) PoC rewards in a boosted hex using HIP-133 rules and would only receive Service Provider boost rewards when it has sufficient ongoing unique offload connections.
+
+## Related Prior HIPs
+
+- [HIP-134](./0134-reward-mobile-carrier-offload-hotspots.md): Rewarded MOBILE Carrier Offload Hotspots
+- [HIP-133](./0133-bridging-gap-for-anti-gaming-measures-phase2.md): Extended CDR requirements for anti-gaming
+- [HIP-84](./0084-service-provider-hex-boosting.md): Created hex boosting rewards## Summary
+
+## Motivation
+
+When a hotspot is placed in a Service Provider boosted hex, it is expected to achieve significant offload. Currently, there are hundreds of hotspots in boosted hexes that earn boosted PoC while selected for offload, yet they have little or no actual offload. These hotspots are essentially farming boosted PoC by manipulating their location and meeting one-time CDR requirements as outlined in HIP-133.
+
+## Stakeholders
+
+ * Wi-Fi Hotspot Deployers: Will receive higher rewards due to reduced gaming of boosted rewards.
+ * Service Providers and the Network Overall: Will gain value from remaining boosted hexes by only paying for PoC when a hotspot is actually offloading data.
+
+
+## Detailed Explanation
+
+Hotspots that assert in a boosted hex:
+
+* Qualify for non-boosted PoC rewards if they meet HIP-133 CDR/disco mapping rules.
+* Qualify for boosted rewards with 25+ unique offload connections over an ongoing 7-day window as specified in HIP134. The hotspot receives boosted rewards for the boosted hex, with this being an ongoing requirement rather than a one-time threshold.
```

Comment:
```suggestion
* Qualify for boosted rewards with 25+ unique offload connections over an ongoing 7-day window as specified in HIP134. The Hotspot receives boosted rewards for the boosted hex, with this being an ongoing requirement rather than a one-time threshold.
```